docs/topics/proc_creating-a-vertx-application.adoc

```diff
@@ -0,0 +1,59 @@
+[id='creating-an-application_{context}']
+= Creating an application
+
+Create a simple `Greeting` application to run on OpenShift using {Vertx}.
+The following procedure shows you how to:
+
+* Write some simple application code that makes use of functionalities provided by {Vertx}.
+* Declare dependencies and configure the application build using a `pom.xml` file.
+* Start your application on localhost and verify that it works.
+
+
+.Prerequisites
+
+* Maven installed.
+* JDK 8 or later installed.
```

Comment:
Now I stumbled upon this -- are we _sure_ that Vert.x (and particularly the version we ship!) supports Java 9 or 10? @cescoffier?